src/shared/components/sign/sign.js

```diff
@@ -33,6 +33,9 @@
       if ($scope.currentUser.school) {
         $scope.currentUser.school = SPF_SINGAPORE_SCHOOLS[$scope.currentUser.school.id];
       }
+
+      // Current year used to calculate age
+      $scope.currentYear = new Date().getFullYear();
     }
```

Comment:
@blueset @SingaporeClouds 

The year of reference should be the current term year, not such the current year. When does it start in Singapore?